backend/api/index.js

```diff
@@ -35,6 +36,20 @@ router.post(
   })
 );
 
+router.get(
+  '/validateLink/:channel',
+  asyncHandler(async (req, res) => {
+    const { channel } = req.params;
+    const isChannelValid = await channelValid(channel);
+
+    if (!isChannelValid) {
+      return res.sendStatus(404).send('Invalid channel');
+    }
+
+    return res.sendStatus(200).send('Valid channel');
```

Comment:
```suggestion
    return res.sendStatus(200).send({ status: 'ok' });
```

Reply:
Thanks for the correction, i really appreciate it 👍 , gonna change it immediately